### ISSUE TITLE
[BUG]Can not download pretrain tokenizer model?

### ISSUE THREAD

**Issue by saber258** [MontrealCorpusTools/Montreal-Forced-Aligner#581]
**Debugging checklist**

[ ] Have you updated to latest MFA version?
I finished MFA download by conda create -n aligner -c conda-forge montreal-forced-aligner today, so I think it's latest version.
[ ] Have you tried rerunning the command with the `--clean` flag?
It's problem in download produres......
**Describe the issue**

when I finished MFA download by conda create -n aligner -c conda-forge montreal-forced-aligner, I download a dictionary and then when I tried to download pretrain tokenzier model, it did not work.

I used command : mfa model download tokenizer japanese_mfa

However, something wrong happend:

usage: mfa model download [-h] [--github_token GITHUB_TOKEN] [--ignore_cache]
                          {acoustic,dictionary,g2p,ivector,language_model} [name]
mfa model download: error: argument model_type: invalid choice: 'tokenizer' (choose from 'acoustic', 'dictionary', 'g2p', 'ivector', 'language_model')

**Comment by mmcauliffe** [comment]
What's the output of `mfa version`?  If you have the latest 2.2.3, it should have tokenizer as a download option, if not I'll try to get 2.2.4 out shortly.

**Comment by saber258** [comment]
Thank you for you reply. I have solved the problem.  I found that my download mfa is at a low version. Thank you for your help!

Here's a little suggestion. The mfa I download in my anthor computer is exactly at a high version and I can use all commands normally. That's why I did not check the mfa version when the problem came out. And I used the same download command in the second computer but generated different result. Is the command "conda create -n aligner -c conda-forge montreal-forced-aligner" can be changed to  automatically download the latest version? I think it may be more convenient.
